src/app/task-displayer/task-displayer.component.ts

```diff
@@ -15,16 +15,26 @@ import { TaskData } from '../taskdata';
   styleUrl: './task-displayer.component.scss'
 })
 export class TaskDisplayerComponent {
-  
+
   taskList$: Observable<TaskData[]> = this.taskService.taskList$;
 
-  
 
-  constructor(private taskService: TaskService) { };
+
+  constructor(private taskService: TaskService) {
+    this.taskService.taskList$.subscribe({
```

Comment:
in order to avoid this, you can subscribe to the observables by using them in the template (html file) with async pipe, it would look like this:

<div *ngIf="taskList$ | async as taskList">
// rest of you code
</div>

This way it will only render data on screen if taskList is preset (and with async pipe it automatically subscribes and unsubscribes)